odoo/tools/mail.py

```diff
@@ -42,6 +42,7 @@
      'data-publish', 'data-id', 'data-res_id', 'data-interval', 'data-member_id', 'data-scroll-background-ratio', 'data-view-id',
      'data-class', 'data-mimetype', 'data-original-src', 'data-original-id', 'data-gl-filter', 'data-quality', 'data-resize-width',
      'data-shape', 'data-shape-colors', 'data-file-name', 'data-original-mimetype',
+     'data-toggle', 'role', 'aria-controls', 'aria-selected',
```

Comment:
At first glance, it seems to make sense, although should not this be for 14.0? :thinking: The tabs snippet already existed at the time I think, there probably are broken sanitized fields with the same problem? Although I just tested on product descriptions in 14.0 (which are sanitized) and I don't have the problem... can you explain why? :thinking: We are probably not talking about the same sanitization... I see here you are in tools/mail.py, how does that impact the sanitization of a field which is displayed on the frontend? :+1: I need more info :+1:

Reply:
After investigation: the description field of courses was transform into an HTML field at https://github.com/odoo/odoo/commit/6d914c005183aa5ae69650e3002a5a85ca83b4a6 but left with full sanitization. It probably should be changed. Will ping the concerned team.